modules/log-collector-resources-scheduling.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-resources-scheduling_{context}"]
+= Configuring resources and scheduling for logging collectors
+
+Administrators can modify the resources or scheduling of the collector by creating a `ClusterLogging` custom resource (CR) that is in the same namespace and has the same name as the `ClusterLogForwarder` CR that it supports.
+
+The applicable stanzas for the `ClusterLogging` CR when using multiple log forwarders in a deployment are `managmentState` and `collection`. All other stanzas are ignored.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo} version 5.8 or newer.
+* You have created a `ClusterLogForwarder` CR.
+
+.Procedure
+
+. Create a `ClusterLogging` CR that supports your existing `ClusterLogForwarder` CR:
++
+.Example `ClusterLogging` CR YAML
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: <name> <1>
+  namespace: <namespace> <2>
+spec:
+  managementState: Managed
+  collection:
+    type: vector
+# ...
```

Comment:
It is better to give an example as below
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogging"
metadata:
  name:  <name> <1>
  namespace: <namespace> <2>
spec:
  managementState: "Managed"
  collection:
    type: "vector"
    tolerations:
    - key: "logging"
      operator: "Exists"
      effect: "NoExecute"
      tolerationSeconds: 6000
    resources:
      limits:
        memory: 1Gi
      requests:
        cpu: 100m
        memory: 1Gi
    nodeSelector:
      collector: needed